### PR TITLE
Swedish translation update

### PR DESCRIPTION
**Pull request by Mr-Bajs**
Buttons and hotkeys

### CODE REVIEW

**Comment by Mr-Bajs** [comment]
@zenseii 
Ready for your review.

File: files/lang/sv.po [hunk 1]
```diff
@@ -3258,7 +3262,9 @@ msgid ""
 "determined by the scenario designer. More difficult maps might include more "
 "or stronger enemies, fewer resources, or other special conditions making "
 "things tougher for the human player."
-msgstr ""
+msgstr "Kartans svårighetsgrad för valda kartan. Kartans svårighets bestäms av kartans kreatör. "
```
[the remark this answers is not in the document]
Comment (Mr-Bajs) [reply files/lang/sv.po]: Changed into "_Svårighetsgraden för den valda kartan._ " to refer to the map two times in the same scentence is redundant and a little bit wierd.

File: files/lang/sv.po [hunk 2]
```diff
@@ -3262,7 +3262,7 @@ msgid ""
 "determined by the scenario designer. More difficult maps might include more "
 "or stronger enemies, fewer resources, or other special conditions making "
 "things tougher for the human player."
-msgstr "Kartans svårighetsgrad för valda kartan. Kartans svårighets bestäms av kartans kreatör. "
+msgstr "Svårighetsgraden för valda kartan. Kartans svårighets bestäms av kartans kreatör. "
```
[the remark this answers is not in the document]
Comment (Mr-Bajs) [reply files/lang/sv.po]: fixed.

File: files/lang/sv.po [hunk 3]
```diff
@@ -2758,7 +2761,7 @@ msgid "Quit"
 msgstr "Avsluta"
 
 msgid "Quit out of Heroes of Might and Magic II."
-msgstr "Avsluta Heroes of Might and Magic II."
+msgstr "Avsluta fheroes2."
```
Comment (ihhub) [inline files/lang/sv.po:2764]: Please revert this change. It was absolutely correct. Players play Heroes of Might and Magic II game powered by fheroes2, not fheroes2 game.

File: files/lang/sv.po [hunk 4]
```diff
@@ -4023,21 +4004,22 @@ msgid "Current game version: "
...
 msgid ""
 "This file was saved for a \"The Price of Loyalty\" map, but the "
 "corresponding game assets have not been provided to the engine."
-msgstr ""
+msgstr "Denna fil är sparad för en The Price of Loyalty-karta men motsvarande speltillgångar "
+"finns inte."
 
 msgid "This saved game is localized to '"
-msgstr ""
+msgstr "Denna spelomgång är översatt till "
```
Comment (ihhub) [inline files/lang/sv.po:4019]: Symbol **'** is missing at the end.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Mr-Bajs , I left 2 comments here. Would you mind to take a look?

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Mr-Bajs , thank you so much for this update!

**Pull request merged**